config/helm/chart/default/tests/Common/operator/deployment-operator_test.yaml

```
@@ -117,7 +117,7 @@ tests:
                   runAsUser: 1001
                   runAsGroup: 1001
                   capabilities:
-                    drop: ["all"]
+                    drop: ["ALL"]
```

```suggestion
                    drop: 
                      - ALL
```
we should be consistent and have it as an actual list everywhere

---

done